index.js

```diff
@@ -2,8 +2,12 @@ const geojsonVT = require('geojson-vt')
 const vtpbf = require('vt-pbf')
 const Logger = require('koop-logger')
 const Winnow = require('winnow')
+const config = require('config')
+const maxAge = config.vectorTiles.maxAge || 3600
 const log = new Logger()
 
+const tilesSetCache = [];
```

Comment:
Shouldn't this be an object if you are going to use as a key-value store?  Also, did you try to implement the koop.cache for this?  I'm happy to pull your branch and give it try if you don't mind.

Reply:
Go for it - I looked at that briefly but was not too sure on the implementation.

Reply:
If you can change `tilesSetCache` to an object instead of an array, I'll merge the PR.  I'll wait to publish a new npm until I see if I can get caching via koop in a different branch/PR, where I will  also add tests, changelog, etc.